src/Sarif.Viewer.VisualStudio/Tags/CaretEventArgs.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft. All rights reserved. 
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+
+namespace Microsoft.Sarif.Viewer.Tags
+{
+    using System;
+
+    internal class CaretEventArgs : EventArgs
```

Comment:
>E [](start = 24, length = 1)

Comment that this is used to send selection events from the caret selection service.

name to => CaretSelectionEventArgs

Reply:
Was resolved without change. Did you decide against the name change?

---
In reply to: [502085227](https://github.com/microsoft/sarif-visualstudio-extension/pull/214#discussion_r502085227) [](ancestors = 502085227)

Reply:
Couldn't decide on a good name... Didn't mean to resolve the comment. It is used for both "caret enter" and "caret leave" event args.

Suggestions?

---
In reply to: [502557583](https://github.com/microsoft/sarif-visualstudio-extension/pull/214#discussion_r502557583) [](ancestors = 502557583,502085227)

Reply:
`CaretSelectionEventArgs`? `CaretChangeEventArgs`?

---
In reply to: [502566200](https://github.com/microsoft/sarif-visualstudio-extension/pull/214#discussion_r502566200) [](ancestors = 502566200,502557583,502085227)